### PR TITLE
Search Catalog URL

### PR DESCRIPTION
**Pull request by nickcoleman**
Documents the URL for: 
- Search Items:   POST /api/products/items/search/
- Search Inventory List:    POST /api/products/items/{uuid}/search/

Adds for Inventory List only, a count of included skus for an item (included_sku_count: <num>).  

The current sku_count property tracks the total number of sku variants for an item.  A user may choose to only include a partial set of item skus in a list.  The inventory list UI will display both both the total and included variants..

### CODE REVIEW

File: Product--SearchCatalog-SubCatalog_and_Inventory.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
 ### URL
 
-###
+Search Items:
+  GET /api/items/<item-uuid>/
 
-??????????
+Search Inventory List:
+  GET /api/product/<item-uuid>/
```
Comment (Gidgidonihah) [inline Product--SearchCatalog-SubCatalog_and_Inventory.md]: I've confused by these endpoints. Why would you be searching with a single item uuid?

File: Product--SearchCatalog-SubCatalog_and_Inventory.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 
 ### URL
 
-###
+Search Items:
+  GET /api/products/items/
 
-??????????
+Search Inventory List:
+  GET /api/products/inventory-lists/<uuid>/
```
Comment (Gidgidonihah) [inline Product--SearchCatalog-SubCatalog_and_Inventory.md]: @jtprince do we need another url for catalogs, or would this need to be named more generically?

Comment (jtprince) [reply Product--SearchCatalog-SubCatalog_and_Inventory.md]: I'm sorry for the confusion--I didn't know this was what you were asking.  There is only one endpoint for catalog searching right now and that is `/api/products/items/search/`  If you want to filter on a particular inventory list or catalog then you need to pass that in as a filter.  cc @nickcoleman

Comment (jtprince) [reply Product--SearchCatalog-SubCatalog_and_Inventory.md]: So, I need to change that endpoint.  It needs to have /search/ on the end to know that elasticsearch should be accessed.

Comment (nickcoleman) [reply Product--SearchCatalog-SubCatalog_and_Inventory.md]: @jtprince  Assume that is the case for the item and search as well?
  GET /api/products/items/search/
  GET /api/products/inventory-list/{uuid}/search/

**Pull request closed without merging**